repos/exporter/lib/handler.ts

```diff
@@ -90,3 +96,6 @@ export const handler: Handler<ExportLambdaBody> = async ({ paths, noc }) => {
 
     console.log(`done ${paths.toString()}`);
 };
+
+export const isBasePeriodTicket = (ticket: WithIds<BaseTicket>): ticket is WithIds<BasePeriodTicket> =>
+    'fareDayEnd' in ticket;
```

Comment:
I'm not sure fareDayEnd is in ticket is it? Also can we inline this function? We're trying to avoid typeguards as it means the compiler can't work out the type for itself